lib/src/components/layouts/StructureEditorLayout/InventoryContainer.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Col } from 'react-bootstrap';
+
+function InventoryContainer({ children }) {
+  return (
+    <Col md={1} sm={2} className="structure-editor__inventory-container">
```

Comment:
🔴  We're moving away from BEM, if this is just semantic use "structure-editor-inventory-container" 👍

Reply:
I didn't know we were moving away from BEM, happy to update this 👍